src/conf.py

```
@@ -127,6 +127,10 @@
 # Enable needs export
 needs_extra_options = ["category", "recommendation", "fls", "decidability", "scope", "release"]
 
+
+# Required guideline fields
+required_guideline_fields = ['category', 'release', 'fls', 'decidability', 'scope', 'tags'] # Id is automatically generated
```

It might be a follow-up PR, but it'd be nice to have ID not be automatically generated and to fail out if that's missing. I think I might have left that configuration to allow for blank IDs up above in this file.

---

Yes, I was thinking the same thing — just by setting `needs_id_required = True`, we can enforce IDs manually.

There’s also another **consistency** issue with id and status:
their validation errors are produced by Sphinx itself before reaching guidelines_checks.py.
So our custom validation logic doesn’t even get a chance to run for those fields.

---

Ah okay. Could you open an issue for each?

---

Alright, i will open it tomorrow.